charts/fluent-operator/values.yaml

```diff
@@ -63,7 +63,24 @@ fluentbit:
       enable: false
       host: "<Elasticsearch url like elasticsearch-logging-data.kubesphere-logging-system.svc>"
       port: 9200
-      logstashPrefix: ks-logstash-log
```

Comment:
`logstashPrefix: ks-logstash-log` shouldn't be commented

Reply:
`logstashPrefix` set to `ks-logstash-log` by default in template file, so I think `logstashPrefix` also should be commented.